CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+## Contributer's Guide
+
+Thanks for your interest in the project! We welcome pull requests from
+developers of all skill levels. To get started, simply fork the master branch
+on GitHub to your personal account and then clone the fork into your
+development environment.
+
+Padmakar Ojha (**dvopsway** on github) is the original creator of the
+SSHmux projet, and currently maintains the project along with Rob Haverkamp (**rjrhaverkamp** on GitHub and Twitter).
+
+Please don't hesitate to reach out if you have any questions, or just need a
+little help getting started.
+
+### Pull Requests
+
+Before submitting a pull request, please ensure you have added or updated tests as appropriate, and that all existing tests still pass with your changes on both Python 2 and Python 3. Please also ensure that your coding style follows PEP 8.
+
+
+### Test coverage
+
+Pull requests won't be merged when test coverage decreases. This helps ensure the quality of the project. To check coverage before submitting a pull request:
+
+```bash
+$ COVERAGE_FILE=.coverage_ssh coverage run tests/test_ssh.py
+$ COVERAGE_FILE=.coverage_validations coverage run tests/test_validations.py
+$ coverage combine .coverage_ssh .coverage_validations
+```
+
+### Code style rules
+
+* always follow the [Google styleguide](http://google.github.io/styleguide/pyguide.html)
+* Docstrings are required for classes, attributes, methods, and functions.
+* Please try to be consistent with the way existing docstrings are formatted. In particular, note the use of single vs. double backticks as follows:
+    * Double backticks
+        * Inline code
+        * Variables
+        * Types
+        * Decorators
+    * Single backticks
+        * Methods
+        * Params
+        * Attributes
+* Format non-trivial comments using your GitHub nick and one of these prefixes:
+    * TODO(rjrhaverkamp): Clean up this part of the code
+    * NOTE(rjrhaverkamp): remeber, 1+1=2
+    * PERF(rjrhaverkamp): blazingly fast!
+* When catching exceptions, name the variable `ex`.
+* Use whitespace to separate logical blocks of code and to improve readability.
+* No single-character variable names except for trivial indexes when looping,
+or in mathematical expressions implementing well-known formulas.
+* Heavily document code that is especially complex and/or clever.
+* When in doubt, optimize for readability.
+
+### Commit Message Format
```

Comment:
Great! I took most of it from https://github.com/falconry/falcon/blob/master/CONTRIBUTING.md though. :)